### ISSUE TITLE
不同的batch_size_kg计算方式 +  valid数据集问题

### ISSUE THREAD

**Issue by yuanyuansiyuan** [xiangwang1223/knowledge_graph_attention_network#35]
1.在load_data.py中，batch_size_kg计算公式如下：self.batch_size_kg = self.n_triples // (self.n_train // self.batch_size)
使用在loader_kgat.py中的def _generate_train_A_batch(self):函数中用于生成kg训练的batch
2.在Main.py中，使用的是args.batch_size_kg， 我看示例parser.py中默认的参数是2048， 运行了示例代码，也是2048
请问batch_size_kg大小为何在两个地方定义不同？

3. 论文中写到会From the training set, we randomly select 10% of interactions as validation set to tune hyper-parameters，代码中似乎没有这部分的处理代码

谢谢！
